CMSgov/mdct-mcr
# MDCT-368 Add scroll to top for route change and submission

## Description
[MDCT-368](https://qmacbis.atlassian.net/browse/MDCT-368). Add scroll to top ability. On route change page will reset to top with no delay or visible movement.

On form submission (admin banner is the only form right now), the movement is visible. (Note there is a delay here as it waits for the write to finish)

### How to test
Run locally. Navigate to the help page and scroll to the bottom. Click on the logo to go to the home. Notice you're at the top. You can toggle between these, scrolling down in between (compare to main).

Submit banner. Wait and see it scroll up.

### Changed Dependencies
none

## Code author checklist

## Reviewer checklist (two different people)
